qml/pages/VideoPlayer.qml

```diff
@@ -0,0 +1,140 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import QtMultimedia 5.6
+import "../components/"
+
+Page {
+    id: videoPage
+
+    // The effective value will be restricted by ApplicationWindow.allowedOrientations
+    allowedOrientations: Orientation.All
+
+    backNavigation: false
+
+    property string videoFile
+    property double controlsOpacity: 1.0
+
+    Rectangle {
+        width: parent.width
```

Comment:
Would it be simpler to use "anchors.fill: parent" in all similar cases when we fill the whole parent?

Reply:
Yes, will do